backend/app/routers/product.py

```
@@ -58,6 +55,21 @@ def post_product(
     return new_product
 
 
+@router.get("/products/me", response_model=List[schemas.ProductOut])
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)